src/containerapp/azext_containerapp/tests/latest/test_containerapp_dapr_resiliency.py

```diff
@@ -0,0 +1,304 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import yaml
+import tempfile
+
+from .common import TEST_LOCATION
+from .utils import create_containerapp_env
+from azext_containerapp.tests.latest.common import (write_test_file, clean_up_test_file)
+
+from azure.cli.testsdk.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, JMESPathCheck)
+
+
+class ContainerappResiliencyTests(ScenarioTest):
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="eastus2")
+    def test_containerapp_resiliency(self, resource_group):
+        self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
+
+        env_name = self.create_random_name(prefix='containerapp-env', length=24)
+        ca_name = self.create_random_name(prefix='containerapp', length=24)
+        resil_name = self.create_random_name(prefix='resil', length=24)
+        bad_resil = "bad-resil"
+        bad_rg = "bad-rg"
+        bad_capp = "bad-capp"
+        resil_policy_count = 1
+        
+        create_containerapp_env(self, env_name, resource_group)
+
+        self.cmd('containerapp create -g {} -n {} --environment {}'.format(resource_group, ca_name, env_name))
+        self.cmd(f'containerapp show -g {resource_group} -n {ca_name}', checks=[JMESPathCheck("properties.provisioningState", "Succeeded")])
+
+        #Incorrect resource group (create)
+        self.cmd('containerapp resiliency create -g {} -n {} --container-app-name {} --cb-interval 15 --cb-consecutive-error 5 --cb-max-ejection 60'.format(bad_rg, resil_name, ca_name), expect_failure=True)
+
+        #Incorrect capp name (create)
+        self.cmd('containerapp resiliency create -g {} -n {} --container-app-name {} --cb-interval 15 --cb-consecutive-error 5 --cb-max-ejection 60'.format(resource_group, resil_name, bad_capp), expect_failure=True)
+
+        #Create app resiliency using flags
+        self.cmd('containerapp resiliency create -g {} -n {} --container-app-name {} --cb-interval 15 --cb-consecutive-error 5 --cb-max-ejection 60'.format(resource_group, resil_name, ca_name))
```

Comment:
Please check the `provisioningState` for these `create/update` commands' result.

Reply:
That property is not available in the payload. It just creates the CRD so the operation should be very quick.